index.html

```diff
@@ -119,11 +119,11 @@ <h2 class="footer-heading">LET'S CONNECT</h2>
             </li>
 
             <li class="footer-link">
-                <a href="omkarpatke1101@gmail.com" class="link-center" target="_blank"><img src="images/mail.svg" alt="email link"></a>
+                <a href="mailto:omkarpatke1101@gmail.com" class="link-center" target="_blank"><img src="images/mail.svg" alt="email link"></a>
```

Comment:
Hello,
@omkarpatke , I have just edited the mail button.
So, I request you to merge this pull request! #1
 
Thank you,
Waiting for you to pull this request!!
@nived-dev
coding@[minds0987](https://github.com/minds0987)
#Stay home #Stay safe